azext_iot/iothub/command_map.py

```diff
@@ -138,6 +139,11 @@ def load_iothub_commands(self, _):
         cmd_group.command("send", "iot_c2d_message_send")
         cmd_group.command("purge", "iot_c2d_message_purge")
 
+    with self.command_group("iot hub state", command_type=iothub_state_ops, is_preview=True) as cmd_group:
```

Comment:
Thoughts on labeling the state command group as experimental instead of preview?